pkg/sidecar/config.go

```diff
@@ -31,6 +31,8 @@ type Config struct {
 	SvidBundleFileNameDeprecated       string `hcl:"svidBundleFileName"`
 	RenewSignal                        string `hcl:"renew_signal"`
 	RenewSignalDeprecated              string `hcl:"renewSignal"`
+	FederatedTrustDomain               string `hcl:"federated_trust_domain"`
```

Comment:
Can there be more then one federated trust domain?

Reply:
good point, I just changed it to be able to include a list of federated trust domains